R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -949,6 +949,19 @@ test_that("fillna() on a DataFrame", {
   expect_identical(expected, actual)
 })
 
+test_that("crosstab() on a DataFrame", {
+  rdd <- lapply(parallelize(sc, 0:3), function(x) {
+    list(paste0("a", x %% 3), paste0("b", x %% 2))
+  })
+  df <- toDF(rdd, list("a", "b"))
+  ct <- crosstab(df, "a", "b")
+  ordered <- ct[order(ct$a_b),]
+  row.names(ordered) <- NULL
+  expected <- data.frame("a_b" = c("a0", "a1", "a2"), "b0" = c(1, 0, 1), "b1" = c(1, 1, 0),
+                         stringsAsFactors = FALSE, row.names = NULL)
+  expect_identical(expected, ordered)
```

Comment:
Ah, we need to update the doc. The behavior changes in https://github.com/apache/spark/commit/6396cc0303ceabea53c4df436ffa50b82b7e233f. We output 0 instead of null. I will send a follow-up PR for this because it also touches Scala and Python code.